### PR TITLE
Update 'Quote' pattern

### PR DESCRIPTION
**Pull request by enriquesanchez**
Updates the 'Quote' pattern, adds an image on top and a separator at the bottom.

Before:

Screen Shot 2020-07-10 at 12 34 37

After:

Screen Shot 2020-07-10 at 12 34 15

### CODE REVIEW

File: lib/patterns/quote.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
...
+	'content'       => "<!-- wp:group -->\n<div class=\"wp-block-group\"><div class=\"wp-block-group__inner-container\"><!-- wp:image {\"width\":125,\"height\":125,\"sizeSlug\":\"large\",\"className\":\"is-style-rounded\"} -->\n<figure class=\"wp-block-image size-large is-resized is-style-rounded\"><img src=\"http://2.gravatar.com/avatar/ea8b076b398ee48b71cfaecf898c582b?s=500&amp;d=mm&amp;r=g\" alt=\"\" width=\"125\" height=\"125\"/></figure>\n<!-- /wp:image -->\n\n<!-- wp:quote {\"align\":\"center\",\"className\":\"is-style-large\"} -->\n<blockquote class=\"wp-block-quote has-text-align-center is-style-large\"><p>" . __( '"I know not all that may be coming, but be it what it will, I\'ll go to it laughing."', 'gutenberg' ) . '</p><cite>' . __( '— Herman Melville, Moby-Dick (1851)', 'gutenberg' ) . "</cite></blockquote>\n<!-- /wp:quote -->\n\n<!-- wp:separator {\"className\":\"is-style-default\"} -->\n<hr class=\"wp-block-separator is-style-default\"/>\n<!-- /wp:separator --></div></div>\n<!-- /wp:group -->",
 	'viewportWidth' => 800,
 	'categories'    => array( 'text' ),
-);
+);
```
Comment (ZebulanStanphill) [inline lib/patterns/quote.php:13]: Looks like there's a missing newline here.

Comment (enriquesanchez) [reply lib/patterns/quote.php]: Oh, good catch! Thank you!

**Comment by kjellr** [comment]
This is cool! Where does the user icon come from? It seems a little inelegant, and I wonder if there's something else we can use.

**Comment by enriquesanchez** [comment]
>This is cool! Where does the user icon come from?

@kjellr I'm pulling it from Gravatar.

>I wonder if there's something else we can use.

Would you suggest a variation of that same idea? Or something totally different?

**Comment by kjellr** [comment]
It would be sort of cool to pull an image of Herman Melville if there is a CC0 one (but I know that might open a can of worms around images we can use here). Alternatively, maybe if there's a G2-style icon we can use instead of this one?

**Comment by enriquesanchez** [comment]
>maybe if there's a G2-style icon we can use instead of this one?

@kjellr Here are a couple of similar graphics using the user icon from G2:

Screen Shot 2020-07-13 at 16 31 21

Screen Shot 2020-07-13 at 16 34 11

**Comment by kjellr** [comment]
I was thinking maybe something like this? I matched the background to the default text color, which makes this a little more bold, too. 

Frame 3

([Here's the SVG I used there](https://cloudup.com/cTkXp-pu0Rf))

**Comment by enriquesanchez** [comment]
That looks great. Will add this graphic to the request already in progress in Trac https://core.trac.wordpress.org/ticket/50650

**Comment by enriquesanchez** [comment]
Updated the avatar image with the new one from the wp.org CDN:

Screen Shot 2020-07-16 at 18 59 57


@kjellr @mapk Would love one last review from you.

**Review by mapk — APPROVED** [review]
LGTM! :shipit:

**Comment by joedolson** [comment]
As a general rule, since the visual use of an HR is the same as its semantic usage, it's semantically correct. There obviously can be cases where a separator isn't really a separator, but usually it's fine.

It's fine for there to be an HR used as a separator; that's not a problem. If it's changeable, then it's really not a big deal, as a user can remove it if they don't want a separation.

The HR represents a separation; if the user wants a visual separation, then that visual separation should also be represented in the page semantics.

I'd actually consider a horizontal rule that creates a visual separation that isn't reflected semantically to be a bigger problem.

We can't dictate semantics to a user, because this block pattern doesn't exist within a context; to judge whether an HR is appropriate or not would be dependent on what other content surrounds the content, and isn't inherent to the HR itself.

**Comment by ZebulanStanphill** [comment]
[(Full discussion in the Slack accessibility channel.)](https://wordpress.slack.com/archives/C02RP4X03/p1595006219446100)

Seeing that this is probably fine (and if not, the potential negative impact is minimal), I'm going to go ahead and merge this. It's simple enough to change later if desired.

**Review by ZebulanStanphill — APPROVED** [review]

**Pull request merged**